src/main/resources/static/api.yaml

```diff
@@ -2602,6 +2662,30 @@ components:
         - username
         - password
         - gender
+    RecoverCmIdRequest:
+      type: object
+      properties:
+        name:
+          type: string
+          example: Raj Sharma
+        gender:
+          $ref: '#/components/schemas/Gender'
+        phoneNumber:
+          type: string
+        yearOfBirth:
+          type: integer
+          example: 2000
+        pmjayId:
```

Comment:
I think you need to send as "Identifier" - { "identifierType" : "PMJAYID", "identifierValue" : "ayushman bharat Id value" }

Reply:
added "PMJAY ID" to unverified identifiers